src/app/app.datatypes.ts

```diff
@@ -41,6 +41,7 @@ export class Transaction {
   outputs: Output[];
   status: boolean;
   timestamp: number;
+  coins_change: number;
```

Comment:
Can we use balance or value instead of coins_change, I feel like this is more in line with the language used so far